### PR TITLE
Be 218: add school income audit view

### PR DESCRIPTION
**Pull request by liumengjun**

### CODE REVIEW

File: server/static/staff/js/school_account/income_audit.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/**
+ * Created by liumengjun on 2016-09-27.
+ */
+$(function(){
+    paginationInit();
+    $('[data-action=switch_status]').click(function(e){
+        var $btn = $(this), $tr = $btn.closest('tr'),
+            status = $btn.attr('status'), recordId = $tr.attr('recordId');
+        var params = {'rid': recordId};
+        if (status != 'yes') {
```
Comment (1e0ng) [inline server/static/staff/js/school_account/income_audit.js:10]: `params['action'] = 'mark_' + status;` and all following code can be simplified.

Comment (liumengjun) [reply server/static/staff/js/school_account/income_audit.js:10]: done

**Comment by 1e0ng** [comment]
At least one unit test (200 test) should be added for the new created url.

**Comment by liumengjun** [comment]
done

**Comment by 1e0ng** [comment]
LGTM

**Pull request merged**